modules/bitcoind.nix

```diff
@@ -209,6 +217,31 @@ in {
         example = "tcp://127.0.0.1:28333";
         description = "ZMQ address for zmqpubrawtx notifications";
       };
+      assumevalid = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "00000000000000000000e5abc3a74fe27dc0ead9c70ea1deb456f11c15fd7bc6";
+        description = "If this block is in the chain assume that it and its ancestors are valid and potentially skip their script verification";
+      };
+      addnode = mkOption {
+        type = types.nullOr types.str;
```

Comment:
Would be nice if this were `addnodes` with type `listOf str`.
The list should be translated into multiple `addnode=` statements.

Reply:
Oh, forgot to fix this. Sure. Added a commit.